### PR TITLE
Make agencies departments again

### PR DESCRIPTION
**Pull request by tobyhede**

### CODE REVIEW

File: db/migrate/20160721013450_make_agencies_departments.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class MakeAgenciesDepartments < ActiveRecord::Migration[5.0]
+  def up
+    sql = "UPDATE sections SET type = 'Department' WHERE type = 'Agency'"
```
Comment (tessereth) [inline db/migrate/20160721013450_make_agencies_departments.rb]: Should *ALL* Agencies be Departments? Are Agencies still a thing?

**Comment by tessereth** [comment]
LGTM

**Comment by ltankey** [comment]
I'm not sure of the context of this PR... Is there an associated card or something?

**Comment by tobyhede** [comment]
@ltankey SITES-417
The issue is that Agencies and Depts are not the same thing. We have been using the word Agency, but the prototype has Depts with many Agencies.

**Comment by micapam** [comment]
In the branch name @ltankey

**Comment by ltankey** [comment]
Ahh sorry missed it in branch name.

@tobyhede @tessereth explained it to me, makes sense, LGTM.

**Comment by tobyhede** [comment]
Sorry, should have carried into the PR title

**Comment by micapam** [comment]
I'm just running this locally to test BTW @tobyhede

**Comment by tobyhede** [comment]
Thanks @micapam Good idea. I was just running it again to double double check

**Comment by micapam** [comment]
Works fine. 
I guess ideally I'd prefer this to be a rake task rather than a migration but I can't think of a scenario in which this would actually be a problem so LGTM.

**Pull request merged**